### PR TITLE
PNTS support + misc. QoL improvements

### PR DESCRIPTION
**Pull request by elfprince13**
Making sure stuff that's only been used on my machine so far is available to enable further development.

Partial progress on #14 (doesn't convert our code, but enables support in client code).

### CODE REVIEW

**Comment by elfprince13** [comment]
Would be great to get this merged into master!

File: pnts.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	def unpackString(self, data, length):
+		self.offset += length
+		return data[self.offset - length : self.offset]
+
+	def unpack(self, fmt, data):
+		calc_len = struct.calcsize(fmt)
+		self.offset += calc_len
+		return struct.unpack(fmt, data[self.offset - calc_len : self.offset])[0]
+
+def main():
```
Comment (KermMartian) [inline pnts.py:132]: Is this idiomatic, vs. just raising the error directly in the conditional below?

File: pnts.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+#--------------------------------------------
+# b3dm.py: Component of GLTF to GLB converter
+# (c) 2016 Geopipe, Inc.
```
Comment (KermMartian) [inline pnts.py:5]: Year is incorrect.

File: pnts.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+#--------------------------------------------
+# b3dm.py: Component of GLTF to GLB converter
```
Comment (KermMartian) [inline pnts.py:4]: Filename is incorrect.

**Review by KermMartian — CHANGES_REQUESTED** [review]
LGTM modulo tiny change requests.

File: pnts.py (hunk 1 above)
Comment (elfprince13) [reply pnts.py:132]: It appears to be idiomatic for this code base. I copied it from your pre-existing code.

https://github.com/Geopipe/gltf2glb/blob/00dc4a652724af01adb25dc44597e81353de49c8/b3dm.py#L126-L131

**Comment by elfprince13** [comment]
@KermMartian - please approve at your convenience.

**Review by KermMartian — APPROVED** [review]

**Pull request merged**